### PR TITLE
[client] Use `.vercel/output` for --prebuilt

### PR DESCRIPTION
**Pull request by TooTallNate**
`.vercel/output` will now be used for the `vc build` output directory, rather than `.output`.

Also removes the `.nft.json` file tracing since it's no longer relevant in the latest `vc build` iteration.

### CODE REVIEW

File: packages/client/src/utils/index.ts [hunk 1]
```diff
@@ -126,18 +126,22 @@ export async function getVercelIgnore(
...
+    const outputDir = posix.join(rootDirectory || '', '.vercel/output');
+    ignores = ['*'];
     const parts = outputDir.split('/');
     parts.forEach((_, i) => {
       const level = parts.slice(0, i + 1).join('/');
       ignores.push(`!${level}`);
     });
     ignores.push(`!${outputDir}/**`);
+
+    // TODO: remove - `package.json` should not be required
```
Comment (EndangeredMassa) [inline packages/client/src/utils/index.ts]: Should this be resolved?

**Review by EndangeredMassa — APPROVED** [review]

File: packages/client/src/utils/index.ts (hunk 1 above)
Comment (styfle) [reply packages/client/src/utils/index.ts]: I dont understand the purpose here 🤔

File: packages/client/src/utils/index.ts [hunk 2]
```diff
@@ -126,18 +126,22 @@ export async function getVercelIgnore(
   prebuilt?: boolean,
   rootDirectory?: string
 ): Promise<{ ig: Ignore; ignores: string[] }> {
-  let ignores: string[] = [];
-
-  const outputDir = posix.join(rootDirectory || '', '.output');
+  const ig = ignore();
```
Comment (styfle) [inline packages/client/src/utils/index.ts:129]: Whats the difference between the `ig` object here vs the `ignores` array?

Comment (TooTallNate) [reply packages/client/src/utils/index.ts:129]: I'm not sure entirely but I didn't change anything related to the response here.

File: packages/client/src/utils/index.ts [hunk 3]
```diff
@@ -126,18 +126,19 @@ export async function getVercelIgnore(
...
+  let ignores: string[];
 
   if (prebuilt) {
-    ignores.push('*');
+    const outputDir = posix.join(rootDirectory || '', '.vercel/output');
```
Comment (styfle) [inline packages/client/src/utils/index.ts:133]:
```suggestion
    const outputDir = posix.join(rootDirectory || '', '.vercel', 'output');
```

**Review by styfle — APPROVED** [review]

File: packages/client/src/utils/index.ts (hunk 3 above)
Comment (TooTallNate) [reply packages/client/src/utils/index.ts:133]: Same thing really 🤷

Comment (styfle) [reply packages/client/src/utils/index.ts:133]: Don't we need it to support Windows?

Comment (styfle) [reply packages/client/src/utils/index.ts:133]: Now that I think about it, the `outputDir.split('/')` on line 135 wont work on windows

Comment (TooTallNate) [reply packages/client/src/utils/index.ts:133]: It's using `posix.join()`

**Pull request merged**